### PR TITLE
harashment people on station

### PR DESCRIPTION
**Pull request by Iamgoofball**
capture

### CODE REVIEW

**Comment by feemjmeem** [comment]
yolomerge immediately

**Comment by ExcessiveUseOfCobblestone** [comment]
fly me to the moon is a great song

**Comment by Slignerd** [comment]
"harashment people on station" is a great line but "to smell their ebola" seems a bit unnecessary

File: strings/traumas.json [hunk 1]
```diff
@@ -62,7 +62,8 @@
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
-        "DON'T EVER TUCH ME"
+        "DON'T EVER TUCH ME",
+		"i was banned for harashment people on station to smell their ebola"
```
Comment (MoreRobustThanYou) [inline strings/traumas.json]: spacing

**Comment by Qustinnus** [comment]
@ExcessiveUseOfCobblestone https://www.youtube.com/watch?v=drGnb30QFwQ&has_verified=1

**Comment by DaxDupont** [comment]
add chumbis

**Comment by Qustinnus** [comment]
our playerbase has enough chumbises in it, we don't need to add more in the code

**Comment by ExcessiveUseOfCobblestone** [comment]
Bayonetta is a great game i've considered being able to summon demons a la bayo as a Hiero boss weapon.

**Comment by epochayur** [comment]
what do you think ebola smells like haha asking for a friend

**Pull request closed without merging**